src/pages/2023-12/next-js-app-router-cache/index.mdx

```diff
@@ -0,0 +1,388 @@
+
+Notice that the parameters of both requests are identical. This is crucial for memoization to work - the `url` and `options` parameter must match across all `fetch` requests.
+
+You might notice that we did not have to add any extra arguments in the `fetch` call to enable memoization. This is because memoization is enabled by default. Here are two reasons:
+
+**1. Performance Optimization:** If different components on the same page are requesting the same data, it's inefficient to fetch that data multiple times.
+
+**2. Consistency Across Components:** If data changes between requests made milliseconds apart, it could lead to inconsistencies in the rendered page. For example, two components displaying user information should show the same data if they are rendered as part of the same request.
+
+##### Opting out
+
+By default, all requests are memoized for performance. We can still opt-out, however. To do this, we simply pass the `signal` property of the `AbortController` to the `options` parameter in `fetch`.
+
+This is different to the typical use of the `AbortController`. Next.js overrides the `fetch` API and we can pass in the `signal` property to the `options` parameter, which tells Next.js that it is a new request, effectively opting out of memoization.
+
+```tsx {1, 4, 5}
+const { signal } = new AbortController();
+async function fetchUserData(userId) {
+  // The `fetch` function is automatically memoized by Next.js during SSR
+  const res = await fetch(`https://api.example.com/users/${userId}`, {
+    signal,
+  });
+  return res.json();
+}
+```
+
+The diagram below demonstrates the step-by-step process for how request memoization works.
+
+![request-memo](/articleAssets/2023-12/next-js-app-router-cache/request-memoization.png)
+
+#### Data Cache
+
+The goal of data cache is to minimize the number of calls made to our origin data source and cache the results across incoming server requests and deployments. Next.js achieves this by overriding the native `fetch` API.
+The `cache` property in the `options` parameter represents the **browser's cache**. In Next.js, the `cache` property represents the **server's data cache**. We will discuss how to configure this soon.
+
+Data cache is useful because if there are multiple requests being made by different users to the same route within about a minute, it makes sense to fetch the JSON response, store it in cache and serve them the cached data.
+
+In this example, we have multiple users visiting our site looking for travel tips about the city of Paris. Our route fetches data from the server like articles, images and recommendations.
+This data does not change often.
+
+When User 1 triggers a `fetch` request, the response is cached inside the data cache and memoized for the same render cycle. This means that the next 4 users who access the same route within a certain period (revalidation period) get served the cached data, resulting in faster response times.
```

Comment:
You mention revalidation period and specific numbers like 4 people or 1 minute but again you have never explained what these concepts are. Revalidation should not be mentioned until you explain what it is.